ccloader/js/ccloader.js

```diff
@@ -257,25 +258,25 @@ export class ModLoader {
 			case 'crosscode':
 				depVersion = this.ccVersion;
 				break;
-			case 'post-game':
-			case 'manlea':
-			case 'ninja-skin':
-				depVersion = this.ccVersion;
-				isExtension = true;
-				break;
 			default:
-				depDesc = 'mod ' + depDesc;
-				mod = mods.find(m => m.name === depName);
-				if (mod) {
-					depVersion = mod.version;
-					enabled = mod.isEnabled;
+				if(KNOWN_EXTENSIONS.indexOf(depName) !== -1) {
```

Comment:
This branch should also check if `depName` refers to an extension from `this.extensions`. i.e. ` this.extensions.indexOf(x) !== 1 || KNOWN_EXTENSIONS.indexOf(x) !== -1`. Oh, and also, I'm sure that the `includes` method is available in the stock nwjs.

Reply:
Ah, okay. I can do that real quick.